packages/core/src/salt-provider/SaltProvider.tsx

```diff
@@ -190,6 +198,11 @@ interface SaltProviderBaseProps {
    * A string. Specifies custom theme name(s) you want to apply, similar to `className`.
    */
   theme?: ThemeName;
+  /**
+   * Either "legacy" or "jpmc".
+   * Specifies branding to be used for the theme.
+   */
+  brand?: BrandName;
```

Comment:
I don't think `jpmc` should appear in Salt code? This is technically jpm not jpmc brand.
Is "legacy" a brand...?

Reply:
np! I just did this as temporary, without worrying too much about prop name